packages/ingest/src/consumer.ts

```diff
@@ -42,9 +44,15 @@ export class Consumer {
   ): Promise<0 | 1> {
     try {
       await fn(this.queueUrl, message);
-      if (this.deleteProcessedMessage) await deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+      if (this.deleteProcessedMessage) {
+        await sqs.deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+      }
       return 1;
     } catch (error) {
+      if (error.code === 'ExecutionAlreadyExists' || error.message.includes('ExecutionAlreadyExists')) {
```

Comment:
Is `error.code` check enough?

Reply:
Yes https://github.com/nasa/cumulus/pull/3389/commits/a1922cd3fefbf0c48439c91e7ca40a875a301e06